google/cloud/storage/google_cloud_cpp_storage.cmake

```diff
@@ -228,7 +228,7 @@ add_library(
     well_known_headers.cc
     well_known_headers.h
     well_known_parameters.cc
-    well_known_parameters.h)
+    well_known_parameters.h internal/generate_message_boundary.cc)
```

Comment:
Sort?

Reply:
Yeah, sorry, I wanted to push the code so we could have the offline discussion